docs/overview/office-add-in-availability.md

```diff
@@ -190,6 +193,7 @@ To work as expected, your Office Add-in might depend on a specific Office host,
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.6</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.7</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.8</a><br>
+        - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.9</a><br>
```

Comment:
In this page ([Host and platform availability](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-in-availability?branch=AlexJ-Excel19)), I notice you didn't add `ExcelApi 1.9` to the list of requirement sets under `Office 2019 for Windows` and `Office 2019 for Mac` -- were these two omissions intentional?

Reply:
@kbrandl Yes, those omissions were intentional. Office 2019 shipped with the July fork, which included up to Excel 1.8. Thanks for checking though.